api/api-doc/storage_service.json

```diff
@@ -920,6 +920,21 @@
             }
          ]
       },
+      {
+         "path":"/storage_service/flush",
+         "operations":[
+            {
+               "method":"POST",
+               "summary":"Flush all memtables in all keyspaces.",
```

Comment:
It is sad that the real explanation why this API exists is hidden in a commit message and not visible to the users. We need to start investing more efforts into documentation (for users) and comments (for developers), than in commit messages :-(

Reply:
The functionality exists forever at the `nodetool` level, the new api is there to allow nodetool/jmx to implement it in a more efficient way.  The api level is not exposed to the end user, but rather to field-engineering and to power-admins.  The `summary` text here can provide some additional information about the call usage, but I'm not sure the motivation for it is necessarily required here.